### DISCUSSION TITLE
Version 1.10.0 RC1

### DISCUSSION THREAD

**Discussion by anuraaga** [open-telemetry/opentelemetry-java#3981] [Q&A]
OpenTelemetry SDK 1.10 will be the first release where the metrics API is considered stable. The Metrics SDK continues to be in development and will be alpha.

We plan to release this RC1 with redundant metrics APIs deprecated, RC2 with them removed, and 1.10.0 the same as RC2 barring unforeseen issues.

### API
* The Metrics API has been merged into the `opentelemetry-api` artifact. `OpenTelemetry.getMeterProvider()` is the new entrypoint
* BREAKING CHANGE: Bound metrics instruments have been removed for now to allow more time to bake while still providing a stable metrics API
* `Double/LongMeasurement.observe` has been renamed to `record`.`observe` is deprecated in this release
* `GlobalMeterProvider` has been deprecated. `GlobalOpenTelemetry.getMeterProvider` should be used instead

### SDK

* The semantic conventions have been updated to 1.8.0

#### Exporter
* The OkHttp gRPC exporters now support experimental retry
* OkHttp dispatcher threads are now spawned as daemon threads
* The JPMS module name for the logs exporter has been fixed
* Metrics exporters can have temporality configured

#### Metrics (Alpha)
* Some user callbacks have been wrapped to catch exceptions instead of throwing
* MinMaxSumCount/Count aggregation has been removed
* Empty aggregator is renamed to `drop`
* Cumulative aggregations will not be aggressively dropped every collection cycle

#### Logs (Alpha)
* LogDataBuilder can now take a SpanContext directly
* SdkLogEmitterProvider.get convenience method added

#### SDK Extensions
* BREAKING CHANGE: Deprecated trace incubator types (DelegatingSpanData, SpanDataBuidler) have been removed
* BREAKING CHANGE: Deprecated `ExecutorServiceSpanProcessor` has been removed
* `cloud.platform` is now populated in AWS `Resource`

## New Contributors
* @svalaskevicius made their first contribution in https://github.com/open-telemetry/opentelemetry-java/pull/3921
* @elevenzhan made their first contribution in https://github.com/open-telemetry/opentelemetry-java/pull/3951

**Full Changelog**: https://github.com/open-telemetry/opentelemetry-java/compare/v1.9.0...v1.10.0-rc.1

<hr /><em>This discussion was created from the release <a href='https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.10.0-rc.1'>Version 1.10.0 RC1</a>.</em>

**Comment by hdost** [comment]
One thing that seems to be true as of a number of versions back is that Stable SDKs are relying on Alpha Metrics. 
I know that this isn't so much of an issue for Metrics Moving forward as the API will be stable as of 1.10.0 full release. However, we should be **very** cautious about this moving into logging.

**Reply by jkwatson** [reply]
They use metrics internally, but do not expose the APIs to end users, so we deemed this to be acceptable and a non-issue.
